src/index.tsx

```diff
@@ -1,14 +1,34 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  HashRouter,
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { PeoplePage } from './pages/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement)
   .render(
-    <Router>
-      <App />
-    </Router>,
+    <HashRouter>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<h1 className="title">Home Page</h1>} />
+          <Route path="home" element={<Navigate to="/" replace />} />
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":selectedSlug?" />
+          </Route>
+          <Route
+            path="*"
+            element={
```

Comment:
In the checklist, it said that you should create a new component with this title (I mean ```<h1 className="title">Page not found</h1>```)